components/nexus-elasticsearch/src/main/java/org/sonatype/nexus/elasticsearch/internal/NodeProvider.java

```diff
@@ -68,6 +71,7 @@ public Node get() {
         this.node = node;
       }
       catch (Exception e) {
+        // TODO: Should this be propagating exception?  ES is non-functional at this point if we get an exception
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)